pkg/internal/checkup/checkup.go

```diff
@@ -257,6 +258,11 @@ func (c *Checkup) createTrafficGeneratorPod(ctx context.Context) error {
 	trafficGeneratorPod := newTrafficGeneratorPod(c.params, secondaryNetworksRequest, pciDevicesVarName)
 
 	log.Printf("Creating traffic generator Pod %s..", ObjectFullName(c.namespace, trafficGeneratorPod.Name))
+	if c.params.Verbose {
+		trafficGeneratorPodJSON, _ := json.Marshal(trafficGeneratorPod)
```

Comment:
Can you use `MarshalIndent`? It makes it much more human readable. 

Also, I know that Marshal basically never returns an error but could you check for it nonetheless?

Reply:
> Can you use MarshalIndent? It makes it much more human readable.

Sure, no problem.

> Also, I know that Marshal basically never returns an error but could you check for it nonetheless?

What do you suggest to do if it fails?

Reply:
come to think of it, I don't think we want MarshalIndent. The log is not that important, and if someone is interested in the prettify version they can copy to an external json tool